src/fetch/index.js

```diff
@@ -13,7 +13,7 @@ export const getFetcher = (which, options) => {
     return which;
   }
 
-  if (!which) which = 'fetch';
+  if (!which) which = 'playwright';
```

Comment:
I don't think we want this! But to get pagination working at all, I had to hardcode this. I'm not sure how this all works still.